.idea/.name

```diff
@@ -1 +1 @@
-Caffeine Project
```

Comment:
yep, not sure if it's a sign of good engineering practice to do this on a public repo.
also, I'm always worried about sensible info and hardcoded path in those config files.

Your call.